environment.yml

```diff
@@ -2,7 +2,7 @@ name: pypeit
 channels:
   - conda-forge
```

Comment:
Please add the `pkgs/main` channel under `conda-forge` to fix `miniforge` issues.

Reply:
thanks for testing! i will do so now...

Reply:
should i specify `pkgs/main` or just use `defaults`?

Reply:
Just tested it and `defaults` also works for me, so I think that `defaults` is good.

Reply:
adding both channels to the list should be good enough for now. if we run into problems, we can specify which channel to use on a per-package basis.